docs/cell_census_storage_and_release_policy.md

```diff
@@ -0,0 +1,82 @@
+# Cell Census storage & release policy
+
+**Status**: Draft.
+
+**Version**: 0.0.1.
+
+**Last edited**: Dec, 2022.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED" "MAY", and "OPTIONAL" in this document are to be interpreted as described in [BCP 14](https://tools.ietf.org/html/bcp14), [RFC2119](https://www.rfc-editor.org/rfc/rfc2119.txt), and [RFC8174](https://www.rfc-editor.org/rfc/rfc8174.txt) when, and only when, they appear in all capitals, as shown here.
+
+-
+
+The cell census MUST be stored in the following S3 root bucket:
+
+`s3://cellxgene-data-public/`
+
+Each release MUST be deposited in a folder under the root that follows this naming convention. `counter` MUST be an integer indicating the release number starting on `1`:
+
+`./cell-census/release-[counter]/`
+
+The Cell Census SOMA-related files MUST be deposited within the release folder on a folder named `soma`:
+
+`./cell-census/release-[counter]/soma/`
+
+All h5ads used to create the cell census MUST be copied within the release folder into a folder named `h5ads`:	
+`./cell-census/release-[counter]/h5ads/`
+
+Any dataset changes, additions, or deletions per release MUST be documented in the following human-readable changelog file name `changelog.txt`:
```

Comment:
Curious of the specific contents in the changelog
By additions/deletions, is this referring to new datasets included for the first time & datasets previously included but no longer?
What level of detail will be captured for 'dataset changes'?

Reply:
Also, asked earlier but added here for clarity -- is this a machine-generated or human-created log?  If the former, what & how is it sourced?